camo/wordbank/answerkey.py

```diff
@@ -0,0 +1,60 @@
+from collections import defaultdict, OrderedDict
+
+def sortWords(hashes):
+    lenmap = defaultdict(list)
+    ordered = []
+    for letter, words in hashes.items():
+        lenmap[len(words)] += letter
+
+    for ln in sorted(lenmap.keys()):
```

Comment:
Can probably do this as a list comprehension.  Something like

`ordered.extend( [lenmap[ln] for ln in sorted(lenmap.keys())])`

That feels less readable to me.  But maybe

```
for ln in sorted(lenmap.keys()):
    ordered.extend(lenmap[ln])
```